src/main/java/teammates/common/datatransfer/FeedbackSessionResultsBundle.java

```diff
@@ -1709,7 +1709,7 @@ public String getQuestionText(String feedbackQuestionId) {
         return sortedMap;
     }
 
-    public LinkedHashMap<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>>
+    public Map<String, Map<FeedbackQuestionAttributes, List<FeedbackResponseAttributes>>>
```

Comment:
Add a Javadoc to state that the method returns a ordered Map with {@code recipientTeam} name as key sorted by {@code compareByTeamQuestionRecipientTeamGiver} comparator.